docs/accessibility.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: v2
+title: Accessibility
+bodyclass: a11y-statement-page
+---
+
+## Accessibility Statement for Leaflet
+
+This is an accessibility statement from Leaflet.
+
+<svg class="a11y-icon" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M0 0h24v24H0z" fill="none"></path><path d="M20.5 6c-2.61.7-5.67 1-8.5 1s-5.89-.3-8.5-1L3 8c1.86.5 4 .83 6 1v13h2v-6h2v6h2V9c2-.17 4.14-.5 6-1l-.5-2zM12 6c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"></path></svg>
+
+### Conformance status
+
+The
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/)
+defines requirements for designers and developers to improve accessibility
+for people with disabilities.
+It defines three levels of conformance:
+Level A, Level AA, and Level AAA.
+
+Leaflet is partially conformant with WCAG 2.1
+[Level A](https://www.w3.org/TR/WCAG21/#cc1).
```

Comment:
This statement if the most important one, and is the main reason (https://github.com/Leaflet/Leaflet/issues/7537) for the accessibility statement.

Thinking out loud:

But it'd be nearly impossible to confidently conclude anything above Level A _for Leaflet as a whole_ at any given point. This is because there are many features that would have to be reviewed, and even if every feature of Leaflet passes Level AA (for example) in isolation, when used together they may fail an SC.

Perhaps - at least in the long run - it'd be better to have a table to state e.g.:

- Basic Leaflet map (no added features) passes Level X.
- Feature A passes level X.
- Feature B passes level X.
- etc.

And since any feature can change over time, they'd have to be re-reviewed, for each version of Leaflet? That's not really feasible...